evals/eval_with_modal.py

```
@@ -0,0 +1,95 @@
+    .copy_local_dir(LOCAL_COGNEE_PATH, "/root/cognee")
+    .run_commands(
+        "cd /root/cognee && poetry install",
+    )
+)
+
+
+@app.function(image=image, gpu="T4", concurrency_limit=5)
+def run_single_repo(instance_data: dict, disable_cognee: bool = False):
+    import subprocess
+    import json
+    import os
```

Ruff (0.8.2)

Redefinition of unused `os` from line 4

Remove definition: `os`

(F811)